### PR TITLE
Added recent post section

### PR DESCRIPTION
**Pull request by srikant96**

### CODE REVIEW

File: blogs/01/index.html [hunk 1]
```diff
@@ -1,4 +1,200 @@
...
+                            aliquid, dicta beatae quia porro id est.
+                        </p>
+                        <div  class="read-more" >
+                            <p><a href="">Read More</a></p>
+                        </div>
+                    </div>  
+                </div>
+
+                <div>
+                    <div><img src= "https://placehold.co/345x200/violet/black" placeholder=""></div>
```
Comment (amezng) [inline blogs/01/index.html:176]: Close the img tag

File: blogs/01/index.html [hunk 2]
```diff
@@ -1,4 +1,200 @@
...
```
Comment (amezng) [inline blogs/01/index.html:156]: close the image tag

File: blogs/01/styles/recent-post.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+}
+section.recent-post>div.container>h2{
+    height: 45px;
+    margin-bottom: 70px;
+    font-size: 32px;
+    font-weight: 400;
+    padding: 0px 15px;
+}
+div.documentation>h3{
+    color: rgb(0, 0, 0);
```
Comment (amezng) [inline blogs/01/styles/recent-post.css:29]: Use hex code for the color

File: blogs/01/styles/recent-post.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+}
+div.read-more>p>a:hover{
+    color: #004eb4;
+}
+div.documentation>p>span{
+    color: black;
+    font-weight: 400;
+}
+div.documentation>div>button{
+    background-color: #8bc34a!important;
```
Comment (amezng) [inline blogs/01/styles/recent-post.css:59]: Don't use `!important`

**Review by amezng — CHANGES_REQUESTED** [review]
MTC

File: blogs/01/styles/recent-post.css (hunk 3 above)
Comment (srikant96) [reply blogs/01/styles/recent-post.css:29]: Done

**Comment by amezng** [comment]
@srikant96 Resolve the conflicts

File: blogs/01/styles/recent-post.css [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    color: white;
+    border: none;
+    margin-top: 40px;
+    border-radius: 5px;
+}
+
+@media screen and (max-width:425px){
+    section.recent-post>div.container{
+        width: 385px;
+        height: 6650px;
```
Comment (amezng) [inline blogs/01/styles/recent-post.css:69]: height is redundant here

File: blogs/01/styles/recent-post.css [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    padding: 0px 15px;
+}
+div.documentation>h3{
+    color:#000;
+    font-size: 24px;
+
+}
+div.read-more{
+    text-decoration: none;
+    color: blue;
```
Comment (amezng) [inline blogs/01/styles/recent-post.css:35]: why using text-decoration none for div?

File: blogs/01/styles/recent-post.css [hunk 7]
```diff
@@ -0,0 +1,76 @@
+section.recent-post{
+    margin-top: 120px;
+}
+
+section.recent-post>div.container{
+    height:2200px;
+    width:1125px;
+    margin:0px auto;
+    background-color: rgb(255, 255, 255);
```
Comment (amezng) [inline blogs/01/styles/recent-post.css:9]: Use hex code for the color code.

**Review by amezng — CHANGES_REQUESTED** [review]
MTC

**Pull request closed without merging**